src/copy.c

```diff
@@ -293,7 +293,7 @@ TSCopyMultiInsertInfoIsFull(TSCopyMultiInsertInfo *miinfo)
 static inline int
 TSCopyMultiInsertBufferFlush(TSCopyMultiInsertInfo *miinfo, TSCopyMultiInsertBuffer *buffer)
 {
-	MemoryContext oldcontext;
+	ExplicitMemoryContext oldcontext;
```

Comment:
Looks like here we can use the macro as well.

Reply:
Same situation as above.

Reply:
Using macro here as well.